homeassistant/components/zwave_js/sensor.py

```diff
@@ -529,6 +535,13 @@ def async_add_sensor(info: ZwaveDiscoveryInfo) -> None:
 
         async_add_entities(entities)
 
+    @callback
+    def async_add_controller_status_sensor() -> None:
+        """Add controller status sensor."""
+        driver = client.driver
+        assert driver is not None  # Driver is ready before platforms are loaded.
```

Comment:
I suggest we move this out one level so we only need to do it once during platform setup.